### ISSUE TITLE
[BUG] Issue with Assistant Bot unable to upload files.

### ISSUE THREAD

**Issue by rrosztoczy** [Azure/AI-in-a-Box#78]
**Solution Accelerators**
This repository contains multiple solution accelerators. Please tell us which ones are involved in your report. (Replace the space in between square brackets with an x)
- [ ] Cognitive Services Landing Zone
- [ ] Semantic Kernel Bot
- [ ] Azure ML Operationalization
- [ ] Edge AI
- [x] Assistant Bot

**Describe the bug**
 I am using the Assistant/bot-in-a-box [project,(https://github.com/Azure/AI-in-a-Box/tree/main/gen-ai/Assistants/bot-in-a-box) When I go to test in the bot in "Test web chat" inside the Azure resource by uploading a sample file (have tried csv, pdf) I only get the following error:

```
The bot encountered an error or bug.

To continue to run this bot, please fix the bot source code.

Object reference not set to an instance of an object.
```

**To Reproduce**
1. Go to "Test in Web Chat" for the bot
2. Attempt to upload a file
3. See error

**Expected behavior**
The file is correctly uploaded using the code interpreter tooling.

**Screenshots**
<img width="878" alt="Screen Shot 2024-03-06 at 4 02 26 PM" src="https://github.com/Azure/AI-in-a-Box/assets/45019878/17e7b4eb-6484-4eac-ac4d-d6d0397e58fc">

**Please complete the following information:**
 - OS: Mac Monterrey 12.5.1
 - Browser Chrome
 - Version 121.0.6167.160 (Official Build) (arm64)

**Additional context**
I am using an unmodified version from the sample code.

**Comment by rrosztoczy** [comment]
Appreciate it Marco! Happy to verify on my end once it is!

**Comment by rrosztoczy** [comment]
Hey @MarcoABCardoso I was curious if you had found the source of the issue and could point me in that direction! I'll probably start investigating tomorrow, but not familiar with the repo or dotnet projects really. I am afraid any progress I could make would be very slow. I have been able to replicate the issue locally, in the test web app, and in a direct line web app.

**Comment by MarcoABCardoso** [comment]
Hello @rrosztoczy !

Thank you for your patience! We have a fix out on the main branch now.
Please let us know if your issue is fixed!

**Comment by rrosztoczy** [comment]
Hi @MarcoABCardoso! Thank you - I was able to successfully upload files using the updated repo! However, when I ask for a download link after updating the files, the bot provides the link text but the link is empty: `<a rel="noopener noreferrer" target="_blank">Download the updated CSV file</a>`.

This is true from various channels using a web app or the test web app.  Do you want me to create a different bug for this one?

**Comment by MarcoABCardoso** [comment]
Hi @rrosztoczy 
I apologize for not seeing this sooner. I don't believe this scenario is currently supported - I've created a feature request as I believe it makes sense! The bot should be able to generate download links for files it creates, which is similar to how it's able to show plots.

**Comment by MarcoABCardoso** [comment]
Closing this as the original issue is addressed, please continue the discussion on #90
